index.html

```diff
@@ -49,6 +70,55 @@
 
       var selectedConfigs = new Set();
 
+      function loadVersions(data) {
+      }
+
+      function navigateToVersion(version) {
+        if(!version) {
+          return // if no version present, do not navigate
+        }
+        let params = new URLSearchParams(document.location.search);
+        params.delete("version");
+        params.append("version", version);
+        url = window.location.pathname + `?${params.toString()}`;
+        window.location.href = url;
```

Comment:
Just wanted to say that this is really clever. I would have tried to reload the data in the table, but instead, by updating the URL, you simply reload the page and re-initialize the page with the correct version `summary.json` automatically. I like it!

Reply:
Thank you so much! However, if we do not want a reload of the browser, we can reload data in the table instead. 
Please Do let me know if you need that change.

Reply:
No, it's good this way! No change requested :)